### PR TITLE
Add Python 3.8 and Python 3.9 scripts

### PR DESCRIPTION
**Pull request by OofChair**
These should be good to go, lmk if you find anything I should fix

### CODE REVIEW

File: scripts/python3.8.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	if [ OS_VERSION == *"ubuntu"* ] || [ OS_VERSION == *"debian"* ]
+	then
+		echo "Detected debian-based install, installing wget"
+		sudo apt install wget make tar
+	else
+		echo "Please install wget, make, and tar then re-run this script!"
+	fi
+fi
+
+echo "Installing Node (LTS Release)"
```
Comment (fvionn) [inline scripts/python3.8.sh:15]: Change Node to Python (3.8)

**Review by fvionn — CHANGES_REQUESTED** [review]
Change Node to Python (3.8)

File: scripts/python3.8.sh (hunk 1 above)
Comment (OofChair) [reply scripts/python3.8.sh]: Fixed

**Comment by OofChair** [comment]
Fixed for both 3.8 and 3.9

**Pull request merged**